atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -81,12 +79,12 @@ export const createCrashDataMap = isTempRecord => {
           // TODO: Validate for integers
           uiType: "text",
         },
-        road_type_id: {
-          label: "Roadway Type",
-          editable: true,
-          uiType: "select",
-          lookupOptions: "atd_txdot__road_type_lkp",
-        },
+        // road_type_id: {
```

Comment:
not finding `road_type_id` anymore?

Reply:
it can be removed. we are dropping this column in the new data model ✌️